src/_shared/js/creatives-single-item.js

```diff
@@ -0,0 +1,11 @@
+import { write } from './dom.js';
```

Comment:
If you are only using this in one place, might be better to save this function in that file, e.g. `manual-single/web/index.js`.

Reply:
All done (by you)